pkg/config/config.go

```diff
@@ -10,16 +10,19 @@ var (
 	MasterKeyPath      string
 	HudsonSecretPath   string
 	CredentialsXmlPath string
+	JsonOutput         bool
 )
 
 const (
 	empty           = ""
 	masterKeyDesc   = "(required) master.key file location"
 	secretDesc      = "(required) hudson.util.Secret file location"
 	credentialsDesc = "(required) credentials.xml file location"
+	jsonDesc        = "(optional) output json"
 	usage           = `Usage:
 
   jenkins-credentials-decryptor \
+    -j \
```

Comment:
I've run the binary with and without the `-j` flag and it looks like some fields are missing on the JSON.  
I don't know what is going on there as I already forgot how this app is structured (been awhile).  
Also would be nice if a field is empty to not be displayed.

I'll take a look on the weekend unless you want to figure it out.

Without the flag:
```shell
make run arg="-m ./test/resources/jenkins_2.141/master.key -s ./test/resources/jenkins_2.141/hudson.util.Secret -c ./test/resources/jenkins_2.141/credentials.xml"
```

With the flag:
```shell
make run arg="-m ./test/resources/jenkins_2.141/master.key -s ./test/resources/jenkins_2.141/hudson.util.Secret -c ./test/resources/jenkins_2.141/credentials.xml -j"
```


![image](https://user-images.githubusercontent.com/3105832/94734271-e2b21f00-0360-11eb-80b9-67435d54c02e.png)